content/en/docs/tasks/observability/distributed-tracing/configurability/telemetry-api/index.md

```diff
@@ -0,0 +1,191 @@
+kind: Telemetry
+metadata:
+  name: mesh-default
+  namespace: istio-system
+spec:
+  tracing:
+  - providers: # only a single tracing provider can be selected at this time
+    - name: zippy-kin
+    customTags:
+      foo:
+        literal:
+          value: bar
+    randomSamplingPercentage: 100
+{{</ text >}}
+
+This configuration overrides the default provider from `MeshConfig`, setting the mesh default to be the `"zippy-kin"`
+provider. It also sets the mesh-wide sampling percentage to be `100`, and configures a tag to be added to all trace
+spans with a name of `foo` and a value of `bar`.
+
+### Configuring namespace-scoped tracing behavior
+
+To tailor the tracing behavior for individual namespaces, add a `Telemetry` resource to the desired namespace. Any tracing
+fields specified in the namespace resource will completely override the inherited field configuration from the configuration hierarchy.
+
+For example:
+
+{{< text yaml >}}
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: namespace-override
+  namespace: myapp
+spec:
+  tracing:
+  - customTags:
+      userId:
+        header:
+          name: userId
+          defaultValue: unknown
+{{</ text >}}
+
+When deployed with into a mesh with the prior mesh-wide example configuration, this will result in
+tracing behavior in the `myapp` namespace that sends trace spans to the `zippy-kin` provider and
+randomly selects requests for tracing at a `100%` rate, but that sets custom tags for each span with
+a name of `userId` and a value taken from the `userId` request header. Importantly, the `foo: bar` tag
+from the parent configuration will not be used in the `myapp` namespace. The custom tags behavior completely
+overrides the behavior configured in the `mesh-default.istio-system` resource.
+
+{{< tip >}}
+Any tracing configuration in a `Telemetry` resource completely overrides configuration of its parent resource in the
+configuration hierarchy. This includes provider selection.
+{{</ tip >}}
+
+### Configuring workload-specific tracing behavior
+
+To tailor the tracing behavior for individual workloads, add a `Telemetry` resource to the desired namespace and use a
+`workload selector`. Any tracing fields specified in the workload-specific resource will completely override the inherited
+field configuration from the configuration hierarchy.
+
+For example:
+
+{{< text yaml >}}
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: workload-override
+  namespace: myapp
+spec:
+  selector:
+    matchLabels:
+      service.istio.io/canonical-name: frontend
+  tracing:
+  - disableSpanReporting: true
+{{</ text >}}
+
+In this case, tracing will be disabled for the `frontend.myapp` workload. Istio will still forward the tracing headers,
+but no spans will be reported to the configured tracing provider.
+
+{{< tip >}}
+It is not valid to have two `Telemetry` resources with workload selectors select the same workload. In those cases,
```

Comment:
Do we alert/warn this to users in any way?

Reply:
Not at this time. As we move beyond `experimental` support, validation / warnings will need to be significantly enhanced.